sdk/python/v1beta1/kubeflow/katib/api/katib_client.py

```diff
@@ -251,8 +254,8 @@ def tune(
             pip_index_url: The PyPI url from which to install Python packages.
             metrics_collector_config: Specify the config of metrics collector, 
                 for example, `metrics_collector_config = {"kind": "Push"}`.
-                Currently, we only support `StdOut` and `Push` metrics collector.
-
+                for using custom metric collectors use "custom_collector" key and provide instance of custom V1Container as value,
+                for example, `metrics_collector_config = {"kind" : "Custom", "custom_collector": <Instance of V1Container>}`.
```

Comment:
Maybe we can reorganize these comments and explain each field in `metrics_collector_config`? Like

```
kind: specify the kind of Metrics Collector (currently we support...)
custom_collector: ...
```